tests/source-mysql/setup.sh

```diff
@@ -4,10 +4,19 @@ set -e
 export TEST_STREAM="estuary_test_$(shuf -zer -n6 {a..z} | tr -d '\0')"
 export RESOURCE="{ \"namespace\": \"test\", \"stream\": \"${TEST_STREAM}\" }"
 
+export MYSQL_DATABASE=test
+export MYSQL_HOST="${MYSQL_HOST:=source-mysql-db-1.flow-test}"
+export MYSQL_PORT="${MYSQL_PORT:=3306}"
+export MYSQL_USER="${MYSQL_USER:=root}"
+export MYSQL_PASSWORD="${MYSQL_PASSWORD:=flow}"
+
+docker compose -f source-mysql/docker-compose.yaml up --detach
+sleep 5
```

Comment:
Would be interesting to try something like [this](https://stackoverflow.com/a/46625302) here.

Reply:
Good idea! Implemented it ✅